functions/src/index.ts

```diff
@@ -43,12 +43,61 @@ app.use(
   }),
 )
 
-import * as firebase from 'firebase/app'
-firebase.initializeApp(firebaseConfig)
+import { initializeApp } from 'firebase/app'
+initializeApp(firebaseConfig)
 
+// Test API
 app.post('/helloWorld', (req: Request, res: Response) => {
   console.log('Hello!')
   res.json({ str: 'Hello World!' })
 })
 
-exports.api = functions.https.onRequest(app)
+import {
+  getAuth,
+  signInWithEmailAndPassword,
+  createUserWithEmailAndPassword,
+} from 'firebase/auth'
+
+const auth = getAuth()
+
+// Login user with email
+// Expects input containing email and password
+app.post('/loginWithEmail', (req: Request, res: Response) => {
```

Comment:
I think we should use doxygen to document the API. It's super easy to use and is pretty standard (you'll notice they did this for the climbing project)

looks like this

```
/**
 * This is a comment.
 *
 * @param name The name of the parameter.
 * @return The return value of the function.
 */
int my_function(const char* name) {
  // Do something.
  return 0;
}
```